assets/scss/components/stencil/cart/_cart.scss

```diff
@@ -504,11 +505,11 @@ $cart-item-label-offset:                $cart-thumbnail-maxWidth + $cart-item-sp
         p {
             // scss-lint:disable ImportantRule
             float: none !important;
-            margin-bottom: spacing("quarter");
+            margin: spacing("third") 0;
             text-align: right;
         }
 
-        form {
+        div {
```

Comment:
We always have a surrounding div around the forms. Verified during unification of the classes/styles. It is safe to go with div.